src/viewport/index.tsx

```diff
@@ -217,13 +217,11 @@ class Viewport extends React.Component<ViewportProps, ViewportState> {
             // VisGeometry.tickIntervalLength, to make it available for use as the length of the
             // scale bar in the UI
             this.visGeometry.handleTrajectoryData(newMsg);
+            
+            simulariumController.scaleBarLabel = createScaleBarLabel(
+                msg.version, this.visGeometry.tickIntervalLength, newMsg.spatialUnits
```

Comment:
Make more readable:
1. Unpack `msg.version` up above and call it `originalVersion` or something
2. Rename newMsg as trajectoryFileInfo